include/deal.II/dofs/dof_accessor.h

```diff
@@ -1265,6 +1265,22 @@ class DoFCellAccessor :  public DoFAccessor<DoFHandlerType::dimension,DoFHandler
   neighbor (const unsigned int) const;
 
   /**
+   * Return the @p ith periodic neighbor as a DoF cell iterator. This function
+   * is needed since the neighbor function of the base class returns a cell
+   * accessor without access to the DoF data.
+   */
+  TriaIterator<DoFCellAccessor<DoFHandlerType, level_dof_access> >
+  periodic_neighbor (const unsigned int) const;
```

Comment:
since you refer to the `@p ith neighbor` above, you should give the function argument a name here. Same in the function below.